SymptomFlowManager.swift

```diff
@@ -0,0 +1,89 @@
+////not sure how?
```

Comment:
I said several times to do a simple 1:1 port (e.g. [here](https://github.com/Co-Epi/app-ios/issues/162#issuecomment-629362297)). E.g:

Kotlin:
```kotlin
class SymptomFlowManager(
    private val symptomRouter: SymptomRouter,
    private val rootNavigation: RootNavigation,
    private val symptomInputsManager: SymptomInputsManager,
    private val reportRepo: TcnReportRepo,
    private val uiNotifier: UINotifier
) {
...
}
```
Swift:
```
class SymptomFlowManager {
    let symptomRouter: SymptomRouter
    let rootNavigation: RootNav
    let symptomInputsManager: SymptomInputsManager
    let reportRepo: TcnReportRepo
    let uiNotifier: UINotifier

    init(symptomRouter: SymptomRouter, rootNavigation: RootNav,
                       symptomInputsManager: SymptomInputsManager, reportRepo: TcnReportRepo, uiNotifier: UINotifier) {

        self.symptomRouter = symptomRouter
        self.rootNavigation = rootNavigation
        self.symptomInputsManager = symptomInputsManager
        self.reportRepo = reportRepo
        self.uiNotifier = uiNotifier
    }

    // ...
}
```
(Obviously commenting/adjusting where needed)